.circleci/config.yml

```diff
@@ -1,35 +1,85 @@
-version: 2
+version: 2.1
+
+orbs:
+  zulip: ponylang/zulip@1
+
 jobs:
-  ponyup:
+  x86-64-unknown-linux-nightly:
     docker:
-      - image: ponylang/ponyc:latest
+      - image: ponylang/ponyc:alpine
     steps:
+      - run: apk add --update py-pip
+      - run: pip install cloudsmith-cli
       - checkout
+      - run: bash .ci-scripts/x86-64-unknown-linux-nightly.bash ${CLOUDSMITH_API_KEY}
+      - zulip/status
 
+  shellcheck-init-script:
+    docker:
+      - image: ponylang/ponyc:latest
+    steps:
+      - checkout
       - run: apt-get update -y && apt-get install -y shellcheck
       - run: shellcheck ponyup-init.sh
```

Comment:
do you want this to alert on failure?

i'd say yes in which case see the zulip/status below that alerts on failure only.